program/src/oracle/upd_aggregate.h

```diff
@@ -266,180 +229,13 @@ static inline void upd_aggregate( pc_price_t *ptr, uint64_t slot )
     }
   }
 
-  // FIXME: MOST OF THE REST OF THE CODE HERE CAN PROBABLY BE REMOVED /
-  // CONDENSED AND/OR MADE MORE EFFICIENT.  AND IT PROBABLY WILL
-  // REPLACED SOON BY THE VWAP MODEL UNDER DEVELOPMENT.  IT IS KEPT HERE
-  // TO KEEP THE CURRENT VWAP CALCULATION AS CLOSE AS POSSIBLE TO WHAT
-  // HOW IT CURRENTLY WORKS.
-
-  uint32_t numa = 0;
-  uint32_t aidx[PC_COMP_SIZE];
-  {
-    int64_t const mprc = agg_price; // FIXME: GRAFT TO NEW CALC TO OLD CALC
-    int64_t const lb = mprc / 5;
-    int64_t const ub = ( mprc > LONG_MAX / 5 ) ? LONG_MAX : mprc * 5;
-
-    for ( uint32_t i = 0; i < numv; ++i ) {
-      uint32_t const idx = vidx[ i ];
-      pc_price_comp_t const *iptr = &ptr->comp_[ idx ];
-      int64_t const prc = iptr->agg_.price_;
-      if ( prc >= lb && prc <= ub ) {
-        uint32_t j = numa++;
-        for( ; j > 0 && ptr->comp_[ aidx[ j - 1 ] ].agg_.price_ > prc; --j ) { // FIXME: O(N^2)
-          aidx[ j ] = aidx[ j - 1 ];
-        }
-        aidx[ j ] = idx;
-      }
-    }
-  }
-
-  // zero quoters
-  ptr->num_qt_ = numa;
-  if ( numa == 0 || numa < ptr->min_pub_ || numa * 2 <= numv ) {
```

Comment:
regarding the test changes, it looks to me like the tests are running with `min_pub_ == 0`. The reason the tests previously gave a result of `status=Unknown` is because of this `numa * 2 <= numv` check, which shouldn't be necessary with the new aggregation algorithm. (The old algorithm ran a prefilter on the active publishers to only include the ones whose price was close to the median price. This check is verifying that the prefilter didn't throw out too many publishers.)

Can you double-check that the tests are running with `min_pub_ == 0`? If they are, then the test changes are ok.